include/exadg/convection_diffusion/preconditioners/multigrid_preconditioner.cpp

```diff
@@ -116,9 +116,36 @@ MultigridPreconditioner<dim, Number>::update()
     this->initialize_mapping();
 
     this->update_matrix_free();
+
+    update_operators_after_grid_motion();
```

Comment:
now we only have one `if(mesh_is_moving)` statement in the multigrid preconditioner (and not spread over two functions as done previously).